cpp/include/cudf/io/parquet.hpp

```diff
@@ -482,6 +488,16 @@ class parquet_writer_options {
    */
   auto get_row_group_size_rows() const { return _row_group_size_rows; }
 
+  /**
+   * @brief Returns target page size, in bytes.
+   */
+  auto get_target_page_size_bytes() const { return _target_page_size_bytes; }
```

Comment:
In some places this is called the "maximum page size" and in other places the "target page size." If pages cannot exceed this limit, I think renaming this to say "maximum" instead of "target" is appropriate.

Reply:
done